packages/darktrace/data_stream/ai_analyst_alert/elasticsearch/ingest_pipeline/default.yml

```
@@ -39,7 +39,8 @@ processors:
       if: ctx.event?.kind == 'alert'
   - set:
       field: event.type
-      value: [info]
+      value: [indicator]
+      if: ctx.event?.category != null && ctx.event.category.contains('threat')
```

This is more of me thinking aloud... perhaps `instanceof Collection` is a more clear check before calling `contains`. 

```suggestion
      if: ctx.event?.category instanceof Collection && ctx.event.category.contains('threat')
```